docs/source/models/gbd2017/causes/causes.rst

```diff
@@ -108,13 +108,41 @@ The Appendix for the GBD 2017 Cause of Death manuscript is also available and Op
 Modeling non-fatal outcomes (Outline)
 ---------------------------
 
-- Figure of process
 .. image:: gbd_nonfatal_flowchart_sim_team.jpg
    :width: 600
 
-**Modeling approach** 
-
-- Modeling tools like DisMod, ST-GPR
+**DisMod** 
+
+Disease Model Meta Regression 2.1 (DisMod, sometimes DisMod MR) is a statistical modeling tool developed for the Global
+Burden of Disease study to estimate non-fatal disease burden. It is the most frequently used tool for non-fatal modeling
+in the GBD and models that are run outside of it are frequently called *custom models*. 
+
+DisMod has a user interface. The 
+`External Version
+<https://vizhub.healthdata.org/epi/>`_
+shows published and final models for GBD 2017.
+The 
+`Internal Version
+<https://internal.ihme.washington.edu/epi/>`_
+is where new and ongoing modeling occurs and is for GBD 2019 (but shows GBD 2017 best models). 
+
+There are three main components to DisMod. The first is that it
```

Comment:
I think it's worth specifying that (if I understand correctly) DisMod estimates each year that it estimates for separately--so it ends up creating excellent age patterns but sometimes rather poor time trends. (Though perhaps the second half of that sentence is unnecessary).

Reply:
Yes, I think this is definitely important to put in somewhere -- including, and perhaps especially, the second half of the sentence because that gives me an idea of some of the limitations of DisMod!